vagrant/provisioning/roles/arkcase-prerequisites/tasks/main.yml

```diff
@@ -64,14 +64,47 @@
       owner: root
       url: https://dl.yarnpkg.com/rpm/yarn.repo
 
-- name: required packages to run ArkCase webapp
+- name: Get yarn version
+  become: yes
+  shell: "yarn -v"
+  register: yarn_version
```

Comment:
I guess the purpose of this merge request is to avoid installing the operating system RPM package, if yarn or node or npm was installed in some other way.

Reply:
Exactly.